src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,4 +87,19 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Formats the person as text, showing all contact details regardless of privacy.
+     */
+    private String getPrintableString(Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
+
+        for (Printable printable : printables) {
+            builder.append(printable.getPrintableString());
```

Comment:
Good job in invoking the new implemented`getPrintableString() `method. On the flip-side, if you don't use it, there is not much pointing implementing it.